keps/sig-storage/1682-csi-driver-skip-permission/README.md

```diff
@@ -182,22 +179,21 @@ A test plan will include the following tests:
 
 ### Rollout, Upgrade and Rollback Planning
 
-_This section must be completed when targeting beta graduation to a release._
-
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g. what if some components will restart
-  in the middle of rollout?
+  No, a rollout should not impact running workloads, since the default behavior
+  remains the same and using this feature requires a change to the CSIDriver.
 
 * **What specific metrics should inform a rollback?**
+  No known rollback criteria.
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and do that now.
+  The upgrade->downgrade->upgrade path should work fine--if the new field is
+  not present in the CSIDriver object, it defaults to current heuristic of
+  looking at fstype and RWO volumes for applying fsgroups.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features,
   APIs, fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  No, this feature does not deprecate any existing functionality.
 
 ### Monitoring requirements
```

Comment:
Looks like this section was missing for beta (apologies for not noticing).  Let's get it filled in https://github.com/kubernetes/enhancements/blame/master/keps/NNNN-kep-template/README.md#L476

I've seen some other storage related PRRs that suggest we have some metrics around CSI drivers: https://github.com/kubernetes/enhancements/pull/2941/files .